unity-client/Assets/UnityGLTF/Scripts/GLTFSceneImporter.cs

```diff
@@ -750,7 +750,7 @@ protected virtual IEnumerator ConstructUnityTexture(byte[] buffer, bool markGpuO
 
 #if !UNITY_EDITOR
             //NOTE(Brian): This breaks importing in editor mode
-            texture.Compress(false);
```

Comment:
Note that this is only to fix the issue for GLTF models, AB models with keep the compression set by the converter. This said, I updated the converter compression to HQ in the last AB converter update.

Reply:
I forgot to mention the branch should be tested with `NO_ASSET_BUNDLES`. In any case this PR will be discarded since the ticket is gonna be marked as not-an-issue.